recipes/dwgsim/meta.yaml

```diff
@@ -1,16 +1,16 @@
 package:
   name: dwgsim
-  version: '1.1.11'
+  version: '1.1.13'
```

Comment:
the version is _actually_ `0.1.13` but it has been incorrect previously.  How should we handle this?

Reply:
Oof, I guess we just keep the incorrect major version number, that really sucks.